hmf-common/src/main/java/com/hartwig/hmftools/common/linx/LinxDataLoader.java

```diff
@@ -217,7 +223,7 @@ private static String generateVisSvDataFilename(@NotNull String basePath, @NotNu
     }
 
     @NotNull
-    private static Map<Integer, Integer> loadSvToCluster(@NotNull String filename) throws IOException
+    private static VisSvData loadSvToCluster(@NotNull String filename) throws IOException
```

Comment:
Minor but I'd rename the input variable to `somaticVisSvDataTsv` since you only support reading that specific file, and think it makes the code more readable now that multiple "vis files" are read